slob/README.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Serialized LOB
+category: Structural
+language: en
+tag:
+  - Object Instantiation
+---
+
+## Intent
+
+* Object models often contain complicated graphs of small objects. Much of the information in these
+  structures isn’t in the objects but in the links between them.
+* Objects don’t have to be persisted as table rows related to each other.
+* Another form of persistence is serialization, where a whole graph of objects is written out as a
+  single large object (LOB) in a table.
+
+## Explanation
+
+* The Forest here represents the object graph.
+* A forest contains animals and plants. As is in real life the forest object contains plants and
+  animals
+  where some animals eat other animals and some eat only plants.
+* These relationships are maintained in the Forest Object.
+* There are 2 types of Serializers available ClobSerializer and BlobSerializer.
+* ClobSerializer uses character or textual based serialization, here the Object graph is converted
+  to XML and then stored as text in DB.
+* BlobSerializer uses binary data for serialization, here the Object graph is converted to Byte
+  Array and then stored as a blob in DB.
```

Comment:
Please follow the pattern template: https://github.com/iluwatar/java-design-patterns/wiki/02.-Pattern-template

See the other patterns for examples

Reply:
Updated as per the template, please review